<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminClass</class>
 <widget class="QWidget" name="AdminClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1473</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администратор</string>
  </property>
  <widget class="QTabWidget" name="tabTables">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1471</width>
     <height>701</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="airports_tab">
    <attribute name="title">
     <string>Аэропорты</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_23">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="airports_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_airports_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_airports_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="days_of_week_tab">
    <attribute name="title">
     <string>Дни недели</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_24">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="days_of_week_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_days_of_week_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_days_of_week_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="schedule_tab">
    <attribute name="title">
     <string>Расписание</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="schedule_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_schedule_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_schedule_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="planes_tab">
    <attribute name="title">
     <string>Самолёты</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_26">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="planes_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_planes_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_planes_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="statuses_tab">
    <attribute name="title">
     <string>Статусы рейсов</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_27">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="statuses_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_statuses_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_statuses_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sales_tab">
    <attribute name="title">
     <string>Ведомость продаж билетов</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_28">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="sales_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_sales_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_sales_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="models_tab">
    <attribute name="title">
     <string>Модели самолётов</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_29">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="models_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_models_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_models_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="flights_tab">
    <attribute name="title">
     <string>Рейсы</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_30">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="flights_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_flights_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_flights_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="benefits_tab">
    <attribute name="title">
     <string>Список льгот</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_31">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1461</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="benefits_layout"/>
    </widget>
    <widget class="QPushButton" name="upd_benefits_table_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Обновить данные в таблице</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save_benefits_table_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>570</y>
       <width>359</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить данные в отчёт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
